addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -82,7 +82,7 @@
 </t>
 
 <t t-name="web.EmojiPicker.searchInput">
-    <input class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active" placeholder="Search for an emoji" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0"/>
+    <input class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active" placeholder="Search for an emoji" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0" t-att-tabindex="ui.isSmall ? -1 : 0"/>
```

Comment:
`ui.isSmall` is not the same as `isMobileOS`.
The autofocus that opens soft keyboard issue is a `isMobileOS` problem, so it should use `isMobileOS` here.

Reply:
Updated

Reply:
`ui.isSmall` indicates that the screen size is small (typically for mobile devices). However, because of this change reducing the screen size on a desktop/laptop for testing may result in different behavior compared to an actual mobile device.

Reply:
Yes it won't cause any issue in that scenario, but it gives false impression that in mobile view the autofocus will be there. But, in reality for mobile devices it won't be there.

Reply:
> Yes it won't cause any issue in that scenario, but it gives false impression that in mobile view the autofocus will be there. But, in reality for mobile devices it won't be there.

This is a rough idea for testing small screen, but for testing actual mobile it should be on mobile phone or the "mobile" mode in browser which I think insert some user agent stuffs to pretend it's mobile device for Odoo.